components/filter/filter-dimension-set-date-time-range-value.js

```diff
@@ -24,6 +25,10 @@ class FilterDimensionSetDateTimeRangeValue extends LocalizeCoreElement(LitElemen
 			 * @type {string}
 			 */
 			endValue: { type: String, attribute: 'end-value' },
+			/**
+			 * @ignore
+			 */
+			inactive: { type: Boolean },
```

Comment:
Currently `filter-tags` just show the `text` of a filter whenever that filter is selected. However, the usage of filter tags for the custom range has the following customizations:
1. We don't want to show the tag if the custom range filter is selected but no date has yet been selected
2. We don't want the `text` to be the `text` of the filter item (by default "Custom date range") but rather something that actually shows the selected date(s).

In order to do this I added two new properties:
1. `inactive` is used in filter in order to filter out this selected filter when we don't want it shown in filter tags (this also has raised the question of if it should trigger a `filter-change` event when selected but no value; my feeling is that it should since there was a change and then consumers can handle this however they wish but I'm open to thoughts on this as it's a bit of a weird situation)
2. `descriptiveText` contains the text to be used in the tag

Reply:
I think this makes sense.

Maybe an alternative to adding the inactive property would be to set the `descriptiveText = null` and the filter tags could be configured to ignore it. But I think I prefer being explicit with the state property. It's a weird state... where the option has been "selected", but from a filtering perspective, it's not _really_ selected. 🤔 

I'm not opposed to `descriptiveText`, but are we confusing the label text with the value text? I hesitate to introduce a property on the value components that's tag-specific... hopefully we don't end up having to do something like that.

Reply:
Currently the list item for the filter uses `value.text` for both the `d2l-list-item` `label` as well as for the displayed list item content. We could use `descriptiveText` in the list item `label` as well, maybe renaming it to something like `description`, in which case it wouldn't be tag specific.

Reply:
I was curious if we should have `valueText`? Does that make sense or no?

Reply:
Maybe - it depends what you were thinking it would be used for? Like would it be the same as what `descriptiveText` is doing here or would it have a larger scope?

Reply:
Yeah, just wondering if that would be more accurate than `descriptiveText`. I'm not against `descriptiveText`... just wondering if there is a better name for it. 🤷‍♂️

Reply:
Yup we can do that